src/patterns/components/masthead/default.hbs

```diff
@@ -39,6 +39,11 @@ hasReactCodeInfo: true
       </a>
     </div>
+    <div class="sprk-c-Masthead__nav-link sprk-o-Stack__item sprk-o-Stack__item--center-column@xxs">
+      <a class="sprk-b-TypeBodyOne sprk-u-TextDecoration--none" href="#nogo">
```

Comment:
We have plain spark link styles and we also have a sprk-c-Masthead__link class. What is this link's design supposed to be on hover? See the other links in the masthead for reference on the class names.

Reply:
We also don't want this link to be text-align right. We want it to be aligned to the right. This causes differences in how the text is rendered when it wraps. I'd suggest using flexbox to align it and not text-align. 
![Screen Shot 2019-06-19 at 9 02 28 AM](https://user-images.githubusercontent.com/2117593/59767819-1dc63700-9271-11e9-8ddf-bb805d94721c.png)